rm-datasource/src/main/java/io/seata/rm/datasource/exec/InsertExecutor.java

```diff
@@ -69,7 +73,7 @@ protected TableRecords beforeImage() throws SQLException {
     @Override
     protected TableRecords afterImage(TableRecords beforeImage) throws SQLException {
         //Pk column exists or PK is just auto generated
-        List<Object> pkValues = containsPK() ? getPkValuesByColumn() : getPkValuesByAuto();
+        List<Object> pkValues = containsPK() ? getPkValuesByColumn() : containsColumns() ? getPkValuesByAuto() : getPkValuesByColumn();
```

Comment:
For readability, it is recommended to add a parenthesis to indicate its priority.

Reply:
fix done